pkgs/by-name/sp/spacebar/package.nix

```
@@ -21,17 +21,17 @@ stdenv.mkDerivation rec {
   ];
 
   installPhase = ''
-    mkdir -p $out/bin
-    mkdir -p $out/share/man/man1/
-    cp ./bin/spacebar $out/bin/spacebar
-    cp ./doc/spacebar.1 $out/share/man/man1/spacebar.1
+    runHook preInstall
+    install -Dm555 ./bin/spacebar $out/bin/spacebar
+    install -Dm644 ./doc/spacebar.1 $out/share/man/man1/spacebar.1
```

```suggestion
    installManpage ./doc/spacebar.1
```

Add `installShellFiles` to `nativeBuildInputs`